cclient/iota_api.c

```diff
@@ -28,6 +36,8 @@ iota_api_result_t iota_api_get_node_info(
 
   result = iota_service_query(service, req_buff, res_buff);
   if (result.error != RC_OK) {
+    log_error(CCLIENT_CORE_LOGGER_ID, "[%s:%d] HTTP request filed. %x\n",
```

Comment:
failed

Reply:
btw, can you create a `HTTP_REQUEST_FAILED = "[%s:%d] HTTP request filed. %x\n"` ?